packages/template/src/index.test.ts

```diff
@@ -0,0 +1,12 @@
+import { helloTemplate } from './template';
```

Comment:
dummy non-web-dev question: What's the educational purpose of this newly added test?

Reply:
I wanted the template to have 80% as the default threshold so i had to add this corny test.